app/models/ask.rb

```diff
@@ -1,5 +1,4 @@
 class Ask < Listing
-  belongs_to :receiver, class_name: "Person", foreign_key: "person_id"
```

Comment:
@maebeale , i don't remember what, but this was causing a problem.

Reply:
k